src/game/scenes/vs.c

```diff
@@ -278,70 +278,80 @@ void vs_render(scene *scene) {
+        dark_green.font = FONT_SMALL;
+        dark_green.cforeground = TEXT_DARK_GREEN;
+
+        text_settings light_green;
+        text_defaults(&light_green);
+        light_green.font = FONT_SMALL;
+        light_green.cforeground = TEXT_GREEN;
+
         // render plug's bitching
         char text[256];
         char money[16];
         fight_stats *fight_stats = &scene->gs->fight_stats;
         snprintf(text, sizeof(text), lang_get(fight_stats->plug_text + PLUG_TEXT_START), fight_stats->sold);
         font_render_wrapped(&font_small, text, 90, 156, 198, COLOR_YELLOW);
 
-        font_render_wrapped(&font_small, "FINANCIAL REPORT", 190, 6, 100, COLOR_GREEN);
+        text_render(&light_green, TEXT_DEFAULT, 190, 6, 140, 6, "FINANCIAL REPORT");
 
-        font_render(&font_small, "WINNINGS:", 190, 16, COLOR_DARK_GREEN);
+        text_render(&dark_green, TEXT_DEFAULT, 190, 16, 90, 6, "WINNINGS:");
         score_format(fight_stats->winnings, money, sizeof(money));
         snprintf(text, sizeof(text), "$ %sK", money);
-        font_render(&font_small, text, 250, 16, COLOR_GREEN);
+        text_render(&light_green, TEXT_DEFAULT, 250, 16, 140, 6, text);
 
-        font_render(&font_small, "BONUSES:", 196, 24, COLOR_DARK_GREEN);
+        text_render(&dark_green, TEXT_DEFAULT, 196, 24, 90, 6, "BONUSES:");
         score_format(fight_stats->bonuses, money, sizeof(money));
         snprintf(text, sizeof(text), "$ %sK", money);
-        font_render(&font_small, text, 250, 24, COLOR_GREEN);
+        text_render(&light_green, TEXT_DEFAULT, 250, 24, 140, 6, text);
 
-        font_render(&font_small, "REPAIR COST:", 172, 32, COLOR_DARK_GREEN);
+        text_render(&dark_green, TEXT_DEFAULT, 172, 32, 90, 6, "REPAIR COST:");
         score_format(fight_stats->repair_cost, money, sizeof(money));
         snprintf(text, sizeof(text), "$ %sK", money);
-        font_render(&font_small, text, 250, 32, COLOR_GREEN);
+        text_render(&light_green, TEXT_DEFAULT, 250, 32, 140, 6, text);
 
-        font_render(&font_small, "PROFIT:", 202, 40, COLOR_DARK_GREEN);
+        text_render(&dark_green, TEXT_DEFAULT, 202, 40, 120, 6, "PROFIT:");
         score_format(fight_stats->profit, money, sizeof(money));
         snprintf(text, sizeof(text), "$ %sK", money);
-        font_render(&font_small, text, 250, 40, COLOR_GREEN);
+        text_render(&light_green, TEXT_DEFAULT, 250, 40, 140, 6, text);
 
-        font_render_wrapped(&font_small, "FIGHT STATISTICS", 210, 60, 60, COLOR_GREEN);
+        text_render(&light_green, TEXT_DEFAULT, 210, 60, 60, 6, "FIGHT STATISTICS");
 
-        font_render(&font_small, "HITS LANDED:", 202, 79, COLOR_DARK_GREEN);
-        snprintf(text, sizeof(text), "%u", fight_stats->hits_landed[0]);
-        font_render(&font_small, text, 276, 79, COLOR_GREEN);
+        text_render(&dark_green, TEXT_DEFAULT, 202, 79, 90, 6, "HITS LANDED:");
+        snprintf(text, sizeof(text), "%u ", fight_stats->hits_landed[0]);
```

Comment:
Here, `"%u "` is used because text_render doesn't render length 1 strings. Need a real fix.

Reply:
Apparently fix is in https://github.com/omf2097/openomf/pull/662